### PR TITLE
UI improvements to Message, Reminder, Recall Board

### PR DESCRIPTION
**Pull request by robertdown**
This moves the UI of the MRR page closer to bootstrap standards, including replacing lots of javascript with Bootstrap's Tab functionality. The PR is ready, though the UI could still be improved even further. The refactoring was more than I anticipated so I want to go ahead and move forward now and make the next rounds of improvements separately. No before and after this time, but a screencapture of the new UI

### CODE REVIEW

File: interface/main/messages/messages.php [hunk 1]
```diff
@@ -179,610 +179,602 @@
...
-                            } else {
-                                $showall = "no";
-                                $lnkvar = "messages.php?show_all=yes&" . $activity_string_html;
-                                $lnkattributes = "name='See All' onclick='top.restoreSession()'";
-                                $otherstuff = "<i id='see-all-tooltip' class='fa fa-users fa-lg text-body' aria-hidden='true'></i>";
-                                $messages = xl('My Messages');
-                            }
+                <ul class="nav nav-pills bg-light" id="main-nav-pills" role="tablist">
+                    <li class="nav-item" id='li-mess' role="presentation">
+                        <a href='#messages-div' class="active nav-link" id='messages-li' data-toggle="pill" role="tab" aria-controls="<?php echo xlt("Message");?>" aria-selected="true"><?php echo xlt('Messages'); ?></a>
```
Comment (bradymiller) [inline interface/main/messages/messages.php:186]: xla for the Message in the aria-controls attirbute

Comment (bradymiller) [reply interface/main/messages/messages.php:186]: also same issue in the codeblocks below

**Comment by bradymiller** [comment]
2DV-2

**Comment by stephenwaite** [comment]
PHGrLWau

**Pull request merged**